pandas/core/frame.py

```diff
@@ -6969,14 +6969,6 @@ def applymap(self, func) -> "DataFrame":
         --------
         DataFrame.apply : Apply a function along input axis of DataFrame.
 
-        Notes
-        -----
-        In the current implementation applymap calls `func` twice on the
```

Comment:
is this no longer accurate?  we regularly get questions about the user-defined functions being called in groupby.apply/agg to determine fast vs slow path

Reply:
In #28854 this was removed from `DataFrame.apply`, and there are comments on the original issue (#28827) saying that the implementation has changed recently, and `DataFrame.applymap` directly calls `DataFrame.apply`